### ISSUE TITLE
Failed to load every video error

### ISSUE THREAD

**Issue by swankwc** [DeltaEvo/GrayjayCrunchyroll#2]
Failed to load video
Plugin [Crunchyrol] [Crunchyrol] Cannot read properties of undefined (reading 'startsWith') (177)[18-19] source.getConten tDetails(. . .)

![Screenshot_20240304-221745](https://github.com/DeltaEvo/GrayjayCrunchyroll/assets/47541224/66f993df-c604-404d-8911-755c02bb707d)

Got some things to play but new stuff is throwing up the following error code. 
Crunchyrol] Unexpeeted token '"ldoctype not valid JSON (1)[0-1]

It might be good to implement something similar to other subscription plugin's.

![Screenshot_20240305-115314](https://github.com/DeltaEvo/GrayjayCrunchyroll/assets/47541224/1f53f2f5-f2be-4fe3-b0a2-2af03a4d0e8a)

**Comment by DeltaEvo** [comment]
Hey, thanks for the report, just to confirm, you used the plugin with a non premium Crunchyroll account?

**Comment by ermnick** [comment]
I get the same error on my non-premium account. I have yet to find a free video to test the source with.
It looks like the way the official sources handle scenarios like this is by using:
```javascript
throw new UnavailableException("Video unplayable")
```

**Comment by ermnick** [comment]
The premium status can probably be extracted from the payload data of the `authenticated_access_token` OAuth bearer token. Maybe the `oauth_scopes` or `benefits` keys in the decoded json object contains that information?

**Comment by swankwc** [comment]
> Hey, thanks for the report, just to confirm, you used the plugin with a non premium Crunchyroll account?

 Yes, I did get a wide range of the non premium content to play but it does started failing across the board again similar to ermnick. Even on things that previously worked.

**Comment by OTKUSteyler** [comment]
I have a similar issue this for me don't want to watch 
Is says there is something wrong with the token.

![Image](https://github.com/user-attachments/assets/ad7cf78a-221a-432c-9290-f6f511628914)
